pkg/composer/restart.go

```diff
@@ -56,7 +56,7 @@ func (c *Composer) Restart(ctx context.Context, opt RestartOptions, services []s
 func (c *Composer) restartContainers(ctx context.Context, containers []containerd.Container, opt RestartOptions) error {
 	var timeoutArg string
 	if opt.Timeout != nil {
-		timeoutArg = fmt.Sprintf("--timeout=%d", *opt.Timeout)
+		timeoutArg = fmt.Sprintf("-t=%d", *opt.Timeout)
```

Comment:
Any reason for this change?

Reply:
compose  stop/restart use ‘timeout’, stop,restart use ‘time’.  all use ‘t’ as shortcut.